lib/include.sh

```diff
@@ -1,8 +1,9 @@
 include() {
+  [[ "$BASHER_INCLUDED_ONCE" == *"$1/$2"* ]] && return 0
```

Comment:
This check can cause problems. Once example case: `user/pack` and `user/package`

Reply:
A possible solution could be to use an array.